src/Reports/TimeReport.php

```diff
@@ -55,7 +55,15 @@ public function __construct(DatePeriod $period, array $projects)
                 if (!$model = ProjectProxy::find($project)) {
                     throw new \Exception(sprintf('%s is not a valid project', $project));
                 }
-                $this->projects[] = $model;            }
```

Comment:
- Why do we need this?
- Will `find()` ever return more than one item?
- We are already iterating through the array of projects, having an additional iteration really smells for me